scripts/backup/pg_backup.sh

```diff
@@ -107,8 +107,7 @@ DB_FILE_GREP=$(gdrive list | grep $FILENAME)
 if [[ $? -ne 0 ]]; then
     echo -e "[ERROR]["$(date +\%Y-\%m-\%d\ %H:%M:%S:%3N)"] Could not recover the FILE_ID ourselves, try running \"gdrive list\" and recover the ID for file \"$FILENAME\" to set pg_backup.config's FILE_ID"
 else
-    echo "Set your pg_backup.config's FILE_ID to this ID:"
+    DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
     FILE_ID="$(echo $DB_FILE_GREP | head -1 | cut -d" " -f1)"
-    echo $FILE_ID
+    sed -i -e 's/^FILE_ID=.*/FILE_ID='"$FILE_ID"'/g' $DIR/pg_backup.config
```

Comment:
@VladimirRyzhkov 
`^FILE_ID=.*` we catch every thing after `FILE_ID` to end line. is it ok?

Reply:
@vladarefiev Yes, sed cares by himself about it. For example, I appended `FOO_ATTR=456` after `FILE_ID=` and then run to set value `foo-123`:
```
# Google Drive backup settings
SERVICE_ACCOUNT_CREDENTIALS=timeless-db-backup.json
TIMEOUT=0
BACKUP_PATH=./
FILE_ID=foo-123
FOO_ATTR=456

######################################

```